examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -89,13 +89,15 @@ volatile sl_status_t callback_status = SL_STATUS_OK;
 int32_t wfx_rsi_get_ap_info(wfx_wifi_scan_result_t * ap)
 {
     sl_status_t status = SL_STATUS_OK;
-    uint8_t rssi       = 0;
+    int32_t rssi       = 0;
     ap->security       = wfx_rsi.sec.security;
     ap->chan           = wfx_rsi.ap_chan;
     memcpy(&ap->bssid[0], &wfx_rsi.ap_mac.octet[0], BSSID_MAX_STR_LEN);
     sl_wifi_get_signal_strength(SL_WIFI_CLIENT_INTERFACE, &rssi);
-    ap->rssi = rssi;
-
+    if (status == SL_STATUS_OK)
```

Comment:
status I don't see status used here other than set at SL_STATUS_OK when we enter the functions. so this check is not too relevant